### ISSUE TITLE
Livro: update Gutenberg Dependency Check to latest version

### ISSUE THREAD

**Issue by mikachan** [Automattic/themes#5802]
We should update the Gutenberg Dependency Check in line with the changes from https://github.com/Automattic/themes/pull/5735, so that any child themes have the same Gutenberg requirement messaging.

**Comment by pbking** [comment]
Does it make sense to try to engineer something like that in core?  A way for a theme to depend on a particular plugin being activated or the error?

**Comment by mikachan** [comment]
> Does it make sense to try to engineer something like that in core? A way for a theme to depend on a particular plugin being activated or the error?

Yeah I think that's a great idea, especially for the Gutenberg plugin. Maybe the dependency on a particular Gutenberg version should be handled in the same way as the minimum required WordPress version for a theme, especially whilst the latest Gutenberg is not part of core. I'm not sure how this currently works, but happy to look into it!

**Comment by jeffikus** [comment]
I'd like to dig into this in our Monday meeting. Will add it to the agenda.
